app/src/main/java/com/sunasterisk/fooddaily/data/source/remote/SearchRemoteDataSource.kt

```diff
@@ -0,0 +1,19 @@
+package com.sunasterisk.fooddaily.data.source.remote
+
+import com.sunasterisk.fooddaily.data.model.FoodDetail
+import com.sunasterisk.fooddaily.data.source.OnLoadedCallback
+import com.sunasterisk.fooddaily.data.source.SearchDataSource
+import com.sunasterisk.fooddaily.data.source.remote.response.GetResponsesAsync
+import com.sunasterisk.fooddaily.data.source.remote.response.SearchResponseHandler
+
+class SearchRemoteDataSource: SearchDataSource.Remote {
```

Comment:
singleton cần private constructor 
```suggestion
class SearchRemoteDataSource private constructor(): SearchDataSource.Remote {
```